routes/chat.js

```diff
@@ -84,6 +84,13 @@ router.post('/chat', async function (req, res) {
       chat: chatId,
     });
 
+    const userSocket = [io.sockets.adapter.rooms.get(req.userId)];
+    const otherSocket = [io.sockets.adapter.rooms.get(otherId)];
+    userSocket.join(chatId.toString());
+    otherSocket.join(chatId.toString());
+
+    io.to(req.userId).to(otherId).emit('chats:created', chat);
```

Comment:
create a function with such functionality in serverSocket.js and export it.
Trying to keep the socket functionality in serverSocket.js and use it elsewhere with encapsulating functions

Reply:
cc5e43e